python-lib/plugin_io_utils.py

```diff
@@ -31,6 +33,7 @@ def unique_list(sequence: List) -> List:
 
     Returns:
        List with unique elements ordered by appearance in the original list
+
     """
     seen = set()
     return [x for x in sequence if not (x in seen or seen.add(x))]
```

Comment:
Agreed it's not super readable. I took this off https://stackoverflow.com/questions/480214/how-do-you-remove-duplicates-from-a-list-whilst-preserving-order because it was a one-liner with many positive marks.
If you feel like this is hard to read, I can replace with a for loop.

Reply:
As it is not a core function, I guess you can leave it as it is